src/index.html

```diff
@@ -1,8 +1,34 @@
 ---
 layout: default.html
 pageTitle: Home
-tags: nav
-order: 1
 ---
+<main>
+    <section class="landing">
+        <img src="https://cdn.glitch.com/3b590aef-5434-4e5f-a2f0-351204103827%2FScreen%20Shot%202020-11-18%20at%2011.33.33%20PM.png?v=1605764026083"
+            alt="">
 
-<p>Add the HTML for your homepage here.</p>
+        <button>
+            <a href="contact">Make Appointment</a>
+        </button>
```

Comment:
This element can look like a button while being an `a` element -- that's valid, but putting an `a` inside an actual `button` element is not (or vice vesa).

```suggestion
            <a href="/contact" class="button">Make Appointment</a>
```

Instead, you can add a class to this link and use that to style this in whatever way you want. 👍🏻